tests/2020-01-01-preview/Microsoft.Peering.Direct.tests.json

```diff
@@ -0,0 +1,61 @@
+{

```

Comment:
Happy to include these tests, but note that it's no longer a requirement going forwards for teams that are onboarded for autogeneration